src/www/ui/search.php

```diff
@@ -101,7 +102,7 @@ function GetResults($Item, $Filename, $tag, $Page, $SizeMin, $SizeMax, $searchty
  
        /* Check upload permission */
        if (!$this->uploadDao->isAccessible($upload_pk, Auth::getGroupId())) {
-        return $UploadtreeRecs;
+        return array($UploadtreeRecs, $totalUploadtreeRecs);
```

Comment:
`$totalUploadtreeRecs` is always `array()`

Reply:
I want to return an array of $UploadtreeRecs and $totalUploadtreeRecs. In case I have missed something, can you please suggest how can I return two variables from a function.